curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/use-closure-to-protect-properties-within-an-object-from-being-modified-externally.english.md

```diff
@@ -24,10 +28,12 @@ Change how <code>weight</code> is declared in the <code>Bird</code> function so
 
 ```yml
 tests:
-  - text: The <code>weight</code> property should be a private variable.
-    testString: assert(!code.match(/this\.weight/g), 'The <code>weight</code> property should be a private variable.');
-  - text: Your code should create a method in <code>Bird</code> called <code>getWeight</code> that returns the <code>weight</code>.
-    testString: assert((new Bird()).getWeight() === 15, 'Your code should create a method in <code>Bird</code> called <code>getWeight</code> that returns the <code>weight</code>.');
+  - text: The <code>weight</code> property should be a private variable and should be assigned the value of <code>15</code>.
+    testString: assert(code.match(/(var|let|const)\s+(weight)\s*\=\s*(15)\;/g), 'The <code>weight</code> property should be a private variable and should be assigned the value of <code>15</code>.');
```

Comment:
The following is a simpler version and allows no semi-colons on end:
```suggestion
    testString: assert(code.match(/(var|let|const)\s+weight\s*\=\s*15\;?/g), 'The <code>weight</code> property should be a private variable and should be assigned the value of <code>15</code>.');
```

Reply:
I thought we intentionally tested for semicolons, is that not the case? Personally, I'm fine with not testing for them, I just want to make sure it is inline with the curriculum.